### PR TITLE
Login Page [WIP]

### PR DESCRIPTION
**Pull request by zafin-hassan**
Added login page. Please review my code, especially styling.

### CODE REVIEW

**Comment by mchlp** [comment]
Is the Login with Google supposed to work or is that still being worked on?

File: pages/login.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        padding: "auto",
+        justifyContent: "space-around",
+        textAlign: "left"
+      }}
+    >
+      <Header3>Log In</Header3>
+      <Header3>Login with your Waterloop Email</Header3>
+      <Header3>
+        <GoogleLogin
+          clientId="404915833701-5kvp9td9jonstfsola74atmkjct4h00d.apps.googleusercontent.com-knrl3gka66fldh83dao2rhgbblmd4un9.apps.googleusercontent.com"
```
Comment (mchlp) [inline pages/login.js]: Double check client ID?

**Comment by kevinbai0** [comment]
Can you update the branch by fetching recent changes from master? It's using a pretty old version of the front-end components so some of the header tags don't match the design system.

**Comment by zafin-hassan** [comment]
> Can you update the branch by fetching recent changes from master? It's using a pretty old version of the front-end components so some of the header tags don't match the design system.

I fetched the changes from master using  $ git fetch origin master. Can you check if its alright now?

**Comment by zafin-hassan** [comment]
> Is the Login with Google supposed to work or is that still being worked on?

Still being worked on

**Pull request merged**